newswires/client/src/App.tsx

```diff
@@ -238,23 +255,49 @@ export function App() {
 										margin-left: 8px;
 									`}
 								>
-									<EuiButton
-										size="s"
-										iconType={'popout'}
-										onClick={() =>
-											window.open(
-												configToUrl({
-													...config,
-													view: 'feed',
-													itemId: undefined,
-												}),
-												'_blank',
-												'popout=true,width=400,height=800,top=200,location=no,menubar=no,toolbar=no',
-											)
-										}
-									>
-										New ticker
-									</EuiButton>
+									<EuiShowFor sizes={['xs', 's']}>
+										<Tooltip
+											tooltipContent={'Open ticker in a new window'}
+											position="left"
+										>
+											<EuiButtonIcon
+												aria-label="New ticker"
+												display="base"
+												size="s"
+												iconType={'popout'}
+												onClick={() =>
+													window.open(
+														configToUrl({
+															...config,
+															view: 'feed',
+															itemId: undefined,
+														}),
+														'_blank',
+														'popout=true,width=400,height=800,top=200,location=no,menubar=no,toolbar=no',
+													)
+												}
+											/>
+										</Tooltip>
+									</EuiShowFor>
+									<EuiShowFor sizes={['m', 'l', 'xl']}>
+										<EuiButton
+											size="s"
+											iconType={'popout'}
+											onClick={() =>
+												window.open(
```

Comment:
Can be moved to a common ticker function.

Reply:
good call, am implementing here: https://github.com/guardian/newswires/pull/279/files